app/models/job_execution.rb

```diff
@@ -106,8 +106,8 @@ def stage
   end
 
   def error!(exception)
-    puts_if_present report_error(exception)
     puts_if_present "JobExecution failed: #{exception.message}"
+    puts_if_present "Error URL: #{report_error(exception)}"
```

Comment:
hmmm if someone does not have rollbar/airbrake this would be `Error URL: ` ?

Reply:
Ah, yes if they don't have a plugin set up then we would return `nil`, though if a plugin does return something I think our current expectation is that it is a URL. Updated the `puts_if_present` logic so we don't show the message if nothing is returned.